SproxydKeysScan/Logging.js

```diff
@@ -0,0 +1,57 @@
+const werelogs = require('werelogs');
+const { env } = require('./index');
+
+const loggerConfig = {
+    logLevel: env.LOG_LEVEL,
+    dumpLevel: env.DUMP_LEVEL,
+};
+
+werelogs.configure(loggerConfig);
+const log = new werelogs.Logger('s3utils:SproxydKeysScan:StatisticsLogger');
+
+class Counter {
+    constructor() {
+        return new Proxy({}, {
+            get(obj, k) {
+                return k in obj ? obj[k] : 0;
+            },
+        });
+    }
+}
+
+class StatisticsLogger extends werelogs.Logger {
+    /**
+     * @constructor
+     * @param {number} logInterval - log summary statistics once every intervalSize seconds
+     */
+    constructor(logInterval) {
+        super();
+        this.interval = logInterval;
+        this.beginTime = new Date(Date.now());
```

Comment:
```suggestion
        this.beginTime = new Date();
```
This should do the same thing.